internal/databases/mysql/backup_list_handler_test.go

```diff
@@ -0,0 +1,119 @@
+package mysql
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var emptyColumnsBackups = []BackupDetail{
```

Comment:
I suggest to also add the creation dates.

Reply:
You mean that need to add the ModifyTime to emptyColumnsBackups, shortValuesBackups and longValuesBackups?

Reply:
> You mean that need to add the ModifyTime to emptyColumnsBackups, shortValuesBackups and longValuesBackups?

Yes